salt/modules/systemd.py

```diff
@@ -973,9 +1013,9 @@ def disable(name, **kwargs):  # pylint: disable=unused-argument
                                        python_shell=False,
                                        ignore_retcode=True) == 0
     return __salt__['cmd.retcode'](
-        _systemctl_cmd('disable', name, systemd_scope=True),
+        _systemctl_cmd('disable', name, systemd_scope=True, no_block=no_block),
         python_shell=False,
-        ignore_recode=True) == 0
```

Comment:
I don't know what you mean by "calling that out".

If you mean "mention it in the PR/commit msg", I don't think that's necessary since I opened https://github.com/saltstack/salt/pull/39633 to fix this in the 2016.3 release branch. I only made the change here because I also added a missing unit test, and I needed this fix here to make the test pass.